modules/govuk_ci/manifests/agent/docker.pp

```diff
@@ -15,9 +15,16 @@
     version => '1.17.1',
   }
 
-  cron::crondotdee { 'docker_system_prune_dangling' :
+  cron::crondotdee { 'docker_system_prune' :
```

Comment:
Can you add a TODO please to remind us to remove this code once the file has been purged.

Reply:
@deanwilson Sure done